### PR TITLE
Feature/jameszhang878/log in page

### LINKED ISSUE #5: Log-In / Create Account page
Create the layout for the login / Create Account page. This should include the logo, section for user input, and the ability for the user to switch between the sign-up and log-in views. The "create account" button should be greyed out until all fields are filled in (like in the designs).

image

### PR DESCRIPTION
**Pull request by Cubified**
### Administrative Info
[GitHub Issue #5]
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Added MAW logo and background vectors
- Added form components for log-in/sign-up flows (i.e. full name, email, password, "keep me signed in," "forgot password," etc.)
- Added function to disable the "create account" button based on missing user input

### Testing
How did you confirm your changes worked? 
- We ran the development server, and the web app worked as specified by the mock-up.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

### CODE REVIEW

**Review by PatrickBrown1 — CHANGES_REQUESTED** [review]
You two did a super good job on your first PR (and like 2 days after I assigned it, serious kudos)! Just a couple of quick changes before I approve.

1. I want to keep the page specific styles separated, so if you could move the logimn specific styles to a new stylesheet, i.e. ./styles/LoginPage.css, that would be awesome (leave the more general styles like color in the index.css). I see you already made the file so just move them over
2. The styling get's a little weird as the screen size starts to shrink. Around 700px width the logo starts to get covered by the login pane.
image
Ask our designers Andrew / Juliette (maybe make a group dm with both) about what to do in this case, but I think something like centering the login pane and moving the logo to the top is a possible idea.
3. Our VP Technology (Justin) added automatic linting to the repo over the weekend and I just merged it in, so your code should (theoretically) not pass the test cases any more due to linting inconsistencies. Update your branch to the current version of development (be super careful with the merge conflicts because he ran the linting, which means every file was changed) and run the command ```npm run lint-fix``` in the frontend directory. This should automatically lint everything it can (and tell you what you need to change manually).

Shoot me a DM and rerequest my review once you wrap these up. Again, super good job with this PR!

**Review by PatrickBrown1 — CHANGES_REQUESTED** [review]
Okay it's super close now, the last thing is that it seems the fixed width constants you are using are too big at small screen sizes (less than ~400px it seems to break down because the form is 300px and the padding is 50px). 

i.e.
image

I think adding one more media query (at say 500px) to make the padding on the form 25px on the sides and the logo size ~300px should fix it. Make sure to check all of the mobile screens to make sure it works (i.e. make sure that the changes I suggested actually work).

**Review by PatrickBrown1 — APPROVED** [review]
Super good job! I'll get one more person to review and merge it in ASAP!

File: frontend/src/pages/LoginPage.js [hunk 1]
```diff
@@ -1,9 +1,72 @@
...
+      <img alt="Make-a-Wish Logo" src="/img/login_logo.svg" className="login_logo" />
+      <form className="login_box" action="/">
+        <div className="login_form">
+          <input
+            placeholder="Full Name"
+            className={isLogin ? "hidden" : ""}
+            onChange={(e) => setName(e.target.value)}
+          />
+          <input placeholder="Email" type="email" onChange={(e) => setEmail(e.target.value)} />
+          <PasswordField
```
Comment (aksharans) [inline frontend/src/pages/LoginPage.js:43]: maybe add className={""} so that its not undefined

File: frontend/src/pages/LoginPage.js [hunk 2]
```diff
@@ -1,9 +1,72 @@
...
+            placeholder="Password"
+            type="password"
```
Comment (aksharans) [inline frontend/src/pages/LoginPage.js:45]: maybe add className="" so its not undefined in PasswordField

**Review by aksharans — COMMENTED** [review]
Left a comment and was seeing a couple linting warnings when running, but other than that LGTM

**Review by aksharans — APPROVED** [review]

**Pull request merged**